dashboard/test/factories.rb

```diff
@@ -1,4 +1,9 @@
 FactoryGirl.define do
+  factory :survey_result do
+    user { create :teacher }
+    properties {{survey2016_ethnicity_asian: "1"}}
+  end
+
```

Comment:
@laurelfan Who actually consumes this?  Am I just creating a sample instance of the data?